Modules/Filtering/LabelMap/test/itkBinaryFillholeImageFilterTest1.cxx

```diff
@@ -20,40 +20,59 @@
 #include "itkSimpleFilterWatcher.h"
 
 #include "itkBinaryFillholeImageFilter.h"
+#include "itkTestingMacros.h"
 
 
 int
 itkBinaryFillholeImageFilterTest1(int argc, char * argv[])
 {
 
-  if (argc != 5)
+  if (argc < 5)
```

Comment:
argc != 5 should cause failure messages.

Reply:
Changed in 8e4315a.